### DISCUSSION TITLE
New pcsx rearmed 13 december

### DISCUSSION THREAD

**Discussion by mattitotti** [OnionUI/Onion#640] [Q&A] [answered]
hello guys i'm new and yesterday i bought an miyoo mini v2 for 80 euro from netherland ( me i'm an italian boy 26 years old) maybe the next maybe Tuesday or wednesday it will arrive, i saw the alpha version 4.1 of onion os and pcsx rearmed the 13 of december has released a new version that fix a lot of problem expecially with the games (t'ai fu, hot pursuite II, grand turismio, crash bash,legend of mana, the audio slowdown of tombi 2 and the relative stutters of the game are fixed, and there is a big chance that they fix Medievil II soundtrack random issue i really hope so...... so please can you update the core?
please responde me soon.
https://www.libretro.com/index.php/state-of-libretro-core-updatesprogress-report-maydecember-2022/

**Comment by schmurtzm** [comment, marked as answer]
Hi, the core is already up to date in 4.1 and already contains the fixes for the games of your list. ;) Current build : a4e249a Nov 15, 2022

**Reply by mattitotti** [reply]
> Hi, the core is already up to date in 4.1 and already contains the fixes for the games of your list. ;) Current build : a4e249a Nov 15, 2022

thanks a lot my friend i really wanna talk with you, this is a big notice for me i hope that they fix medievil II soundtrack problems, and i have another question there is duckstation core in onion os or swanstation? if there isn't will be in the future a port in onion os?
thanks for respond me
